lib/img/local.js

```diff
@@ -43,6 +43,22 @@ LocalDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function localDelete(key) {
+    return new P(function deletePromise(resolve, reject) {
```

Comment:
style nit: use of a named function for the closure here is inconsistent with other files in this repo; dunno whether you care or not but it was odd enough that I noticed :-)